### ISSUE TITLE
[BD-46] alpha: Incorrect color for Bubble component 

### ISSUE THREAD

**Issue by PKulkoRaccoonGang** [openedx/paragon#2672]
Incorrect color for Bubble component (variant="warning")

![image](https://github.com/openedx/paragon/assets/93188219/f3b5464a-a95b-4a4a-8767-a7b601a2cb34)

**Comment by PKulkoRaccoonGang** [comment]
We changed the color of the Bubble to optimize accessibility. After these changes, the black color of the elements of the yellow dough is as expected.

PRs with accessibility optimization:
- https://github.com/edx/brand-edx.org/pull/44
- https://github.com/openedx/paragon/issues/1428
